package.json

```diff
@@ -125,10 +127,19 @@
     "start:prod": "REACT_APP_ENV=prod node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom",
-    "lint:fix": "eslint --fix --",
+    "lint": "eslint .",
```

Comment:
do we need this? dont think we ever just run it without the --fix flag?

Reply:
I add this because I think sometime engineer want to learn what they can do it better, otherwise the --fix always fix it for them. I myself used it for learning. but we can also remove it here if you prefer and expect engineer will use `eslint .` to debug.